### ISSUE TITLE
Improvements

### ISSUE THREAD

**Issue by lucianomunoz** [MontagueMGM/shape_keys_plus#1]
Hey! man, really great addon, you should advertise it a bit more.
2 things.

well one is something that maybe you're struggling with, any way for it to completely replace the original Shapekeys Panel by hiding the other one or such?.
the second one, is that it would be nice that since you can select shapekeys there would be a command to make a custom property on a designated bone and driver connect each property with each shapekey selected, and also maybe even set some settings for the connections beforehand, that would aid so much so many workflows!
thanks, if you can't do any of the above i totally understand, but i just wanted to let you know that there are people out there using it! and loving it!

**Comment by MontagueMGM** [comment]
Thanks for the feedback. I originally thought that the default Shape Keys panel couldn't be hidden but I may have just found a way to accomplish it. I'll see what I can do about the improvements for shape key selection.

**Comment by lucianomunoz** [comment]
oh that would be totally great!, glad to hear from you!
thanks for listening to your users =)

**Comment by MontagueMGM** [comment]
It's going to take a couple of days to clean up the code before I start making improvements. This was my first add-on and I wasn't thinking much about manageability and PEP-8 at the time.

I might need an example .blend file with the type of workflow you were mentioning in the original post with properties and shape keys. Just to be sure, but I'm guessing you're talking about using a property on a bone to drive multiple shape keys at once (and in a different way for each shape key), like with facial expressions?

**Comment by lucianomunoz** [comment]
No, my idea was given a certain number of shapekeys that you select, the function could create 1 custom attribute per each shapekey and connect it via driver directly to it.

**Comment by MontagueMGM** [comment]
Which types of pre-connection settings do you have in mind for the custom properties?

**Comment by lucianomunoz** [comment]
Like a one click solution to create custom props in the specified bone, and
connect them to the selected shapekeys via drivers.

**Comment by lucianomunoz** [comment]
Thats something that has to be done for rigs that will be linked to an
other fil even if you don want to create a fancy interface, thats the only
way to expose these attributes

**Comment by MontagueMGM** [comment]
Looks like such a feature is too complicated for me to work on at the moment. I'll have to get back to it in the future. For now I'll publish v1.1.0 with just the updated UI and the ability to hide the default panel.

**Comment by lucianomunoz** [comment]
Great man!, no problem!

**Comment by MontagueMGM** [comment]
v1.2.0 is released with an attempt at the custom property feature you suggested, located in Tool Shelf >> Create >> Shape Keys+. Were you referring to the fact that shape keys can't be edited on linked rigs unless driven by a proxy?
